### ISSUE TITLE
Windows Implementation

### ISSUE THREAD

**Issue by raflitp** [hustvl/4DGaussians#158]
Hi I'm very interested in trying 4DGS, but after I read some of the tutorials I found that the training seems to use linux. Is there any other options to do it on windows ??

**Comment by guanjunwu** [comment]
Hi, windows can also run the 4DGS, do you have any problem?

**Comment by raflitp** [comment]
> Hi, windows can also run the 4DGS, do you have any problem?
> (You can use ChatGPT to transform `xxx.sh` to `xxx.bat` files format.

Yeah I can't install the dependencies like depth-diff-gaussian-rasterization and simple-knn

**Comment by abnerLeeqinghao** [comment]
May I ask if it was successful? I couldn't use windows to compile depth-diff-gaussian-rasterization and simple-knn. Eventually, I chose WSL2
